src/libs/actions/Session.js

```diff
@@ -234,6 +237,29 @@ function signIn(password, twoFactorAuthCode) {
         });
 }
 
+function signInWithShortLivedToken(accountID, email, shortLivedToken, encryptedAuthToken, exitTo) {
+    Onyx.merge(ONYXKEYS.ACCOUNT, {...CONST.DEFAULT_ACCOUNT_DATA, loading: true});
+
+    createTemporaryLogin(shortLivedToken, encryptedAuthToken).then((response) => {
```

Comment:
Is createTemporaryLogin missing the email param? Maybe it's not really needed in `API.CreateLogin`, but NAB we don't have to fix that in this PR